src/infrastructure/entitys/alert.ts

```diff
@@ -0,0 +1,43 @@
+import { Entity, Column, PrimaryColumn, PrimaryGeneratedColumn, CreateDateColumn, Generated } from 'typeorm';
+import { MPPAlert } from '../../core/Alert';
+
```

Comment:
The use of those decorators makes the typescript compiler scream, you need to add the key `strictPropertyInitialization` to `tsconfig.json` with the value of `false`

Reply:
כן, הוספתי אבל זה לא הפסיק לצרוח